ragna/core/_authentication.py

```diff
@@ -49,10 +49,11 @@ class RagnaDemoAuthentication(Authentication):
     def __init__(self) -> None:
         self._password = os.environ.get("RAGNA_DEMO_AUTHENTICATION_PASSWORD")
 
-    _JWT_SECRET = secrets.token_urlsafe(32)
+    _JWT_SECRET = os.environ.get(
+        "RAGNA_DEMO_AUTHENTICATION_SECRET", secrets.token_urlsafe(32)[:32]
```

Comment:
I might have approved a bit too fast :) 
Why is there a truncation over the last 32 chars only on ` secrets.token_urlsafe(32)[:32] ` ? 
Was it maybe supposed to be something like this ?
```python 
os.environ.get(
        "RAGNA_DEMO_AUTHENTICATION_SECRET", secrets.token_urlsafe(32)
)[:32]
```

Reply:
The number passed to `token_urlsafe` is the number of bytes to generate. This is greater than the number of characters:

```pycon
>>> import secrets
>>> len(secrets.token_urlsafe(32))
43
```

Meaning, if we want 32 characters for the token, we have to slice the returned secret.

Reply:
Ok got it, its a matter of number of bytes that can result from 1 to 3 chars : 
https://docs.python.org/fr/3.8/library/secrets.html